pkg/controller/clusterdeployment/clusterdeployment_controller_test.go

```diff
@@ -197,7 +197,7 @@ func TestClusterDeploymentReconcile(t *testing.T) {
 				testSecret(pullSecretSecret, pullSecretKey, "{}"),
 				testSecret(sshKeySecret, adminSSHKeySecretKey, "fakesshkey"),
 			},
-			validate: func(c client.Client) {
+			validate: func(c client.Client, t *testing.T) {
```

Comment:
In this delete case, validate that the job was deleted as well (add one in existing objects)

Reply:
Good catch. I couldn't add here as this just triggers a re-sync but it seems to fit above in the more straightforward delete test.